GLPT_Cocoa.inc

```diff
@@ -1,89 +1,390 @@
+
+procedure NSShow (obj: id);
```

Comment:
I would suggest moving this function to glptError as well. We could use a {$DEFINE SYS_LOG} in this function to write to toggle the system log on all platforms at compile time. See https://www.freepascal.org/~michael/articles/eventlog/eventlog.pdf